autogen/docs/behaviorspace.md.mustache

```diff
@@ -47,7 +64,7 @@ synchronization.
 BehaviorSpace offers you a much better way to solve this problem. If you specify
 a subset of values from the ranges of each slider, it will run the model with
 each possible combination of those values and, during each model run, record the
-results. In doing so, it samples the model's parameter space -- not
+results. (Since NetLogo 6.4 it is possible to specify a non-exhaustive set of combinations of values.) In doing so, it samples the model's parameter space -- not
```

Comment:
I'm not sure why this needs to be parenthetical, I think it can just be a regular sentence.  Also spacing doesn't match the surrounding block.

Reply:
I wound up rewriting the paragraph. If you don't think it is clearer I will just get rid of the parentheses